pytest_splinter/plugin.py

```diff
@@ -501,9 +501,9 @@ def get_browser(splinter_webdriver, retry_count=3):
                 raise
 
     def prepare_browser(request, parent, retry_count=3):
-        splinter_webdriver = request.getfuncargvalue('splinter_webdriver')
-        splinter_session_scoped_browser = request.getfuncargvalue('splinter_session_scoped_browser')
-        splinter_close_browser = request.getfuncargvalue('splinter_close_browser')
+        splinter_webdriver = request.getfixturevalue('splinter_webdriver')
```

Comment:
please update minimum pytest requirements for that this works

Reply:
@bubenkoff is a `pytest`->`pytest>=XXX` change in setup.py enough or would I miss something then?

Reply:
I've specified `pytest>=3.0.0` in setup.py for now.